backend/src/main/java/de/sambam/gamenighttracker/security/AppUser.java

```diff
@@ -0,0 +1,22 @@
+package de.sambam.gamenighttracker.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Document(collection = "appusers")
```

Comment:
is there a difference between `appusers`,  `gameUser` and `user` ? Maybe simplify the namespace...

Reply:
i was not sure if I could use the general user object for the authentication, while the AppUser object has only 2 properties... the Player object is basically only a property of the GameSession object.

Reply:
I could not tell the names apart, although i have seen your code several times now. If you find more appropriate names, everything will be fine.